src/ucp/core/ucp_ep.h

```diff
@@ -245,6 +246,13 @@ typedef struct ucp_ep_config {
          * (currently it's only AM based). */
         const ucp_proto_t   *proto;
     } stream;
+    
+    struct{
+        /* Protocols used for am operations
+         * (currently it's only AM operations). */
+        const ucp_proto_t *proto;
+    } am_u;
```

Comment:
What does "u" mean?

Reply:
user, because the user is defining the am's. I would just use am but it is already a member of the struct for something else.